network/handlers/drain_test.go

```diff
@@ -74,6 +74,11 @@ func TestDrainMechanics(t *testing.T) {
 		}
 	)
 
+	const (
+		timeout   = 100 * time.Millisecond
+		razorThin = timeout - time.Nanosecond
```

Comment:
I think having `epilon = ` here and `timeout-epsilon` below would probably be clearest, assuming the type system allows it.

Reply:
ok done.